### PR TITLE
Fix link on Storage Classes page

### LINKED ISSUE #35740: Broken link on Storage Classes page
**Problem:**
The text [**Storage Policy Based Management for dynamic provisioning of volumes**](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/policy-based-mgmt.html) on [link](https://kubernetes.io/docs/concepts/storage/storage-classes/) is not working, it should point to these https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/master/documentation/policy-based-mgmt.md
Check the page [Storage Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/) for the problem.

**Page to Update:**
https://kubernetes.io/docs/concepts/storage/storage-classes/

**Proposed Solution:**
It should resolve this issue by adding a simple link and enhancing the document line.
Update the link on the text **Storage Policy Based Management for dynamic provisioning of volumes** to https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/master/documentation/policy-based-mgmt.md and not to this [link](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/policy-based-mgmt.html).

/kind document
/assign
/language en

### PR DESCRIPTION
**Pull request by kadtendulkar**
Fixes #35740

Updated page [Storage Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/)

### CODE REVIEW

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -543,7 +543,7 @@ The following examples use the VMware Cloud Provider (vCP) StorageClass provisio
         persistent volume (virtual disk) is being created. The virtual disk is
         distributed across the Virtual SAN datastore to meet the requirements.
 
-        You can see [Storage Policy Based Management for dynamic provisioning of volumes](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/policy-based-mgmt.html)
+        You can see [Storage Policy Based Management for dynamic provisioning of volumes](https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/master/documentation/policy-based-mgmt.md)
```
Comment (sftim) [inline content/en/docs/concepts/storage/storage-classes.md:546]: Even though that repo is archived, I'd go for:
```suggestion
        You can see [Storage Policy Based Management for dynamic provisioning of volumes](https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/fa4c8b8ad46a85b6555d715dd9d27ff69839df53/documentation/policy-based-mgmt.md)
```
as this is even more resistant to drift.

**Review by sftim — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/storage/storage-classes.md (hunk 1 above)
Comment (reylejano) [reply content/en/docs/concepts/storage/storage-classes.md:546]: @kadtendulkar , please see sftim's [suggestion](https://github.com/kubernetes/website/pull/35741#discussion_r939548272)

**Pull request merged**